js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,85 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options,
+    });
+
+    this.settings = app.settings.clone();
+    this.localSettings = app.localSettings.clone();
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toJSON()));
+    this.listenTo(this.localSettings, 'sync',
+      () => app.localSettings.set(this.localSettings.toJSON()));
+  }
+
+  get events() {
+    return {
+      'click .js-smtpContainer input[type="reset"]': 'resetSMTPFields',
+    };
+  }
+
+  resetSMTPFields() {
+    this.settings.set('smtpSettings',
+      this.settings.get('smtpSettings').defaults(), { validate: true });
+    this.render();
+  }
+
+  getFormData(subset = this.$formFields) {
+    return super.getFormData(subset);
+  }
+
+  save() {
+    this.localSettings.set(this.getFormData(this.$localFields), { validate: true });
```

Comment:
Could you add this line as the first line in this function?

```
this.trigger('saving');
```

Sorry... forgot to include that when I gave you the code earlier. Without it, the "Settings Saved." status bar message at the bottom of the screen never goes away.